src/vm-vcpu/src/vcpu/mod.rs

```diff
@@ -138,7 +140,8 @@ impl KvmVcpu {
     /// Configure MSRs.
     fn configure_msrs(&self) -> Result<()> {
         let entry_vec = msrs::create_boot_msr_entries();
-        let msrs = Msrs::from_entries(&entry_vec);
+        // This unwrap is safe because we are creating valid kvm_msrs entries.
+        let msrs = Msrs::from_entries(&entry_vec).unwrap();
```

Comment:
Since we are not checking in any way that `create_boot_msr_entries()` won't create a vec with more than KVM_MAX_MSR_ENTRIES elements, we could stay on the safe side here and propagate the error. Does it sound too exaggerated? :-?

Reply:
Right now the function where we get the msrs from is creating a static vector of a known size. If the function `create_boot_msr_entries` ever creates more than 256 entries, I think we would be dealing with a programming error, in which case it is better to immediately exit. I think the guideline is that programming errors should be handled with `panic!`.

This is though a library code, so we can do some extra diligence. How about this course of action instead:
- `create_boot_msr_entries` returns `Msrs` instead of a Vec which is converted to `Msrs` afterwards
- inside the function it's safe to call `Msrs::from_entries(&entry_vec).unwrap()` because the function is the one that creates the `entry_vec` and it knows its exact size